### PR TITLE
1.7.0-beta8

### PR DESCRIPTION
**Pull request by shiftkey**
## Overview

 - [x] approval on release notes
 - [x] publish to beta

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.7.0-beta8": [
+      "[Fixed] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```
Comment (iAmWillShepherd) [inline changelog.json]: The PR this was fixed in is #7508.

Comment (iAmWillShepherd) [reply changelog.json]: Also, should this be [Added] instead of [Fixed]???

Comment (outofambit) [reply changelog.json]: ohh yeah those are both good points

Comment (outofambit) [reply changelog.json]: actually the issue linked there is the original issue
```suggestion
      "[Added] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.7.0-beta8": [
+      "[Added] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
+      "[Fixed] Pressing \"Shift\" + \"Alt\" in Commit summary moves input-focus to app menu - #6366. Thanks @AndreiMaga!",
+      "[Fixed] Incorrectly encoding URLs affects issue filtering - #7506",
+      "[Improved] Command line interface warns with helpful message when given a remote URL - #7452. Thanks @msztech!",
+      "[Improved] Lowercase pronoun in \"Revert this commit\" menu item - #7534",
+      "[Improved] Performance improvements to \"Pull Requests\" list - #7501",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] \"Pull Requests\" list reflects pull requests from GitHub more quickly - #7501",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.7.0-beta8": [
+      "[Added] Added accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Added] Accelerator and access key to \"Exit\" menu item - #6507. Thanks @AndreiMaga!",
```

File: changelog.json (hunk 1 above)
Comment (iAmWillShepherd) [reply changelog.json]: Yea, I know, figured the PR was better since we are giving someone a shoutout.

**Pull request merged**